### PR TITLE
Fetch all entries to transform

### LINKED ISSUE #82: [Bug] Only first 100 entries are transformed
Hello 👋 

I’m running into an issue when using the `transformEntries()` method on a large number of entries. Only the first 100 entries are transformed. I assume this is due to the paginated response from the management API with a default limit of 100.

@stefanjudis has pointed me to https://github.com/contentful/migration-cli/blob/master/src/lib/fetcher.ts#L26-L29, so I'll give it a shot and make a PR.

### PR DESCRIPTION
**Pull request by connor-baer**
Fixes #82.

### CODE REVIEW

File: src/lib/fetcher.ts [hunk 1]
```diff
@@ -23,12 +24,24 @@ export default class Fetcher implements APIFetcher {
...
+    let skip: number = 0
+
+    while (true) {
+      const response = await this.makeRequest({
+        method: 'GET',
+        url: `/entries?sys.contentType.sys.id[in]=${ids.join(',')}&skip=${skip}`
+      })
+
+      entries = entries.concat(response.items)
+      skip += 100
```
Comment (TimBeyer) [inline src/lib/fetcher.ts]: I'd prefer not to have a magic number here and use `response.items.length`.

Comment (connor-baer) [reply src/lib/fetcher.ts]: Good point!

**Comment by connor-baer** [comment]
@TimBeyer I'll add some tests after lunch.

**Comment by Khaledgarbaya** [comment]
LGTM although at some point we will change the code to not have all entries in memory at once.
@zcei I think we can merge this.

**Comment by Khaledgarbaya** [comment]
@connor-baer Thanks for the PR we'll release a `next` version soon

**Pull request merged**